pkg/services/datasources/errors.go

```diff
@@ -11,4 +15,6 @@ var (
 	ErrDataSourceFailedGenerateUniqueUid = errors.New("failed to generate unique datasource ID")
 	ErrDataSourceIdentifierNotSet        = errors.New("unique identifier and org id are needed to be able to get or delete a datasource")
 	ErrDatasourceIsReadOnly              = errors.New("data source is readonly, can only be updated from configuration")
```

Comment:
I am not a fan that now there are two different ways of defining errors but I understand the reasoning so it's okay to keep it like this.

Reply:
ya I agree, but don't wanna club unrelated change in this PR.